### ISSUE TITLE
release `@react-icons/all-files`

### ISSUE THREAD

**Issue by maddhruv** [react-icons/react-icons#593]
**Describe the bug**
Looks like `@react-icons/all-files` hasn't been released lately with `react-icons` so I am missing the newly added icons packs

**Minimal sample repository**
Not Required

**Expected behavior**
For example Tabler Icons are missing from `all-files`, Can you please do a release with latest `react-icons`

**Screenshots**

**Desktop (please complete the following information):**
 - Device: [e.g. PC, iPhone6]

**Additional context**

**Comment by maddhruv** [comment]
@kamijin-fanta

**Comment by SukkaW** [comment]
@maddhruv I believe it is caused by too many files generated, causing npm to refuse to host the package.

**Comment by maddhruv** [comment]
I think by `npm pack` and npm publish --tarball` using the tarball you would be able to publish, can you please try

**Comment by maddhruv** [comment]
Hey @nolanleung looks like with the latest release as well, the `all-files` package is not updated. Can you look into this?

**Comment by SukkaW** [comment]
> I tried to publish, but it was rejected due to too many files. I think I need to plan on splitting the package.

One possible solution is to split the ESM and CJS packages, which could reduce the number of files to half.

**Comment by kamijin-fanta** [comment]
I don't think that is a good solution. Currently the package contains 80,000 files, which is causing problems such as long installation times.

**Comment by codsane** [comment]
> I don't think that is a good solution. Currently the package contains 80,000 files, which is causing problems such as long installation times. Also, the number of icons will continue to increase.

While it may defeat the purpose of having a package named "all-files", maybe the icons could be split up into packages based on their icon library? So, a `@react-icons/fa` package, a `@react-icons/hi` package, and so-on?

**Comment by fregante** [comment]
Those packages already exist but they don't contain the separate icons. The main setup should just be fixed to export single files in each package and then you should deprecate this duplicate "all-icons" package.

**Comment by Wolverin-e** [comment]
To all the people still finding a workaround, we can also do: [npm install](https://docs.npmjs.com/cli/v8/commands/npm-install) [link_to_gzipped_tarball](https://github.com/react-icons/react-icons/releases).

For an Example:
```sh
$ npm i https://github.com/react-icons/react-icons/releases/download/v4.7.1/react-icons-all-files-4.7.1.tgz
```

This also adds a dependency in the package.json file.

References:
- https://docs.npmjs.com/cli/v8/commands/npm-install
- https://github.com/react-icons/react-icons/releases (<- goto this page for latest releases)

**Comment by AkashMehta27** [comment]
> To all the people still finding a workaround, we can also do: [npm install](https://docs.npmjs.com/cli/v8/commands/npm-install) [link_to_gzipped_tarball](https://github.com/react-icons/react-icons/releases).
> 
> For an Example:
> 
> ```shell
> $ npm i https://github.com/react-icons/react-icons/releases/download/v4.7.1/react-icons-all-files-4.7.1.tgz
> ```
> 
> This also adds a dependency in the package.json file.
> 
> References:
> 
> * https://docs.npmjs.com/cli/v8/commands/npm-install
> * https://github.com/react-icons/react-icons/releases (<- goto this page for latest releases)

I am facing a weird issue here.
Somehow this file just gets stuck on download.

This download error is only happening for the "react-icons-all-files....tgz" for all releases.

Just to test it out, even if I try to manually download this file from Github, the same thing happens, it just gets stuck on downloading and then eventually times out. 
 
Tried different release versions as well, same thing. All other files that are not a "react-icons-all-files....tgz" file, download perfectly fine.

Any thoughts?

**Comment by stephenasuncionDEV** [comment]
When is this being updated?
